integration_tests/src/utils.rs

```diff
@@ -1104,3 +1098,68 @@ pub async fn populate_routers_eth(rita_identities: InstanceData) {
     info!("Sending 50 eth to all routers");
     send_eth_bulk((ONE_ETH * 50).into(), &to_top_up, &web3).await;
 }
+
+pub async fn add_exits_contract_exit_list(db_addr: Address, rita_identities: InstanceData) {
+    let web3 = Web3::new("http://localhost:8545", WEB3_TIMEOUT);
+    let miner_private_key: clarity::PrivateKey = MINER_PRIVATE_KEY.parse().unwrap();
+    let miner_pub_key = miner_private_key.to_address();
+
+    let res = add_exit_admin(
+        &web3,
+        db_addr,
+        miner_pub_key,
+        miner_private_key,
+        None,
+        vec![],
+    )
+    .await
+    .unwrap();
+
+    web3.wait_for_transaction(res, TX_TIMEOUT, None)
+        .await
+        .unwrap();
+
+    let cluster = TEST_EXIT_DETAILS.get("test").unwrap();
+    let nonce = web3.eth_get_transaction_count(miner_pub_key).await.unwrap();
+
+    for (i, id) in rita_identities.exit_identities.iter().enumerate() {
+        let mut mesh_v2 = id.mesh_ip;
+        for exit in cluster.instances.values() {
```

Comment:
Is this trying to reconcile the exit id's in the contract with the configured exit cluster data? We should remove exit cluster data from the exit config and have the exit look at the contract for that.

Reply:
This setups the correct exit identities in the contract with mesh_ip_v2 instead of mesh_ip (which is returned by spawn_rita). I removed the exit cluster list from exit settings. There are some naming issues in the test since I hardcoded orginal exits with a cluster structure, i was planning to move it to just a vector of exits in my next PR

Reply:
@ptulugu  why not update spawn_rita to return mesh_ip_v2? or both ip's go ahead and move to the vector of exits in this pr.